docs/products/clickhouse/get-started.md

```diff
@@ -5,77 +5,486 @@ sidebar_label: Get started
+      cloud_name              = "google-europe-west1"
+      plan                    = "startup-16"
+      service_name            = "my-clickhouse"
+      maintenance_window_dow  = "sunday"
+      maintenance_window_time = "22:00:00"
+      termination_protection  = true
+
+      clickhouse_user_config {
+        service_log = true
+      }
+    }
+
+    output "clickhouse_service_host" {
+      value = aiven_clickhouse.clickhouse.service_host
+    }
+    output "clickhouse_service_port" {
+      value = aiven_clickhouse.clickhouse.service_port
+    }
+    output "clickhouse_service_username" {
+      value = aiven_clickhouse.clickhouse.service_username
+    }
+    output "clickhouse_service_password" {
+      value     = aiven_clickhouse.clickhouse.service_password
+      sensitive = true
+    }
+    ```
 
-1.  In the **Create ClickHouse database** window, enter a name for your
-    database and select **Create database**.
+1. Run `terraform plan` > `terraform apply --auto-approve`.
 
 </TabItem>
-<TabItem value="2" label="Aiven API">
+<TabItem value="3" label="K8s">
+1. Update file `example.yaml`:
 
-Call the
-[ServiceClickHouseDatabaseCreate](https://api.aiven.io/doc/#tag/Service:_ClickHouse/operation/ServiceClickHouseDatabaseCreate)
-endpoint to create a database.
+   - Add `service_log: true` and `terminationProtection: true`.
+   - Update `maintenanceWindowDow: sunday` and `maintenanceWindowTime: 22:00:00`.
 
+   ```yaml
+   apiVersion: aiven.io/v1alpha1
+   kind: Clickhouse
+   metadata:
+     name: my-clickhouse
+   spec:
+     authSecretRef:
+       name: aiven-token
+       key: token
+
+     connInfoSecretTarget:
+       name: my-clickhouse-connection
+       annotations:
+         foo: bar
+       labels:
+         baz: egg
+
+     tags:
+       env: test
+       instance: foo
+
+     userConfig:
+       service_log: true
+
+     project: my-aiven-project
+     cloudName: google-europe-west1
+     plan: startup-16
+
+     maintenanceWindowDow: sunday
+     maintenanceWindowTime: 22:00:00
+     terminationProtection: true
+     ```
+
+1. Update the service by applying the configuration:
+
+   ```go
+   kubectl apply -f example.yaml
+   ```
+
+1. Review the resource you updated with the following command:
+
+   ```go
+   kubectl get clickhouses my-clickhouse
+   ```
+
+The resource can stay in the `REBUILDING` state for a couple of minutes. Once the state
+changes to `RUNNING`, you are ready to access it.
 </TabItem>
 </Tabs>
 
-## Connect to ClickHouse
+See the available configuration options in:
+
+- [Aiven Operator for Kubernetes®: ClickHouse](https://aiven.github.io/aiven-operator/api-reference/clickhouse.html)
+- [Advanced parameters for Aiven for ClickHouse®](/docs/products/clickhouse/reference/advanced-params).
 
-1.  Get the latest Docker image of [the ClickHouse client from Docker
-    Hub](https://hub.docker.com/r/clickhouse/clickhouse-client)
+## Connect to the service{#connect-to-service}
```

Comment:
for me this paragraph looks strange. It is basically about where to find the credentials. It looks like it should be a part of the previous paragraph

Reply:
This section is to demonstrate various ways of connecting to a service, one of them being the use of Aiven Console. It says you need to go to "Quick Connect" and follow the instructions provided for a selected language, which ultimately results in connecting to the service.
It's not the part of the previous paragraph, which demonstrates something different - how to configure the service.

Reply:
But if you want to show the ways to connect, then the tabs should be different clickhouse clients, not different ways to setup clickhouse in aiven.
1. Http (play console)
2. Docker/cli
3. k8s
4. writing query in aiven console
5. ...

Reply:
Thank you :-) The purpose of this article is to show sample use cases and examples of what's possible, not to demonstrate details for all the possible scenarios. The idea is to take the user to "QuickConnect", where they can see possible options, select the preferable scenario and proceed accordingly. We trust them they can follow up without detailed instructions covering all the options and we provide an example only.